### PR TITLE
Add placeholder attribute

### PR DESCRIPTION
**Pull request by asantoz**
This PR introduces the placeholder attribute on the Autocomplete component.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class Autocomplete extends Component {
 		onDelete: PropTypes.func,
 		onFocus: PropTypes.func,
 		onBlur: PropTypes.func,
-		onChange: PropTypes.func,
+		placeholder: PropTypes.string,
```
Comment (raphaguasta) [inline src/index.js]: We need to remove `onChange: PropTypes.func`?

Comment (asantoz) [reply src/index.js]: My bad :smile: fixed!

**Comment by raphaguasta** [comment]
@stavarengo Ca we merge it? 🤘

**Comment by stavarengo** [comment]
Yes of course @raphaguasta .

**Pull request merged**